great_expectations/cli/example.py

```diff
@@ -498,6 +557,25 @@ def _check_aws_env_vars() -> set[str]:
     return result
 
 
+def _check_aws_rds_env_vars() -> set[str]:
```

Comment:
Just thinking out loud - Could this use `_check_aws_env_vars()` instead? Just to avoid duplication, it looks like the same code. I think the more general name is ok `_check_aws_env_vars()` since it is checking for credentials that allow connection to all AWS resources (not just RDS).
Also see Thu's comment on this method: https://github.com/great-expectations/great_expectations/pull/8230/files#r1251178799